src/util-runmodes.c

```diff
@@ -266,6 +280,17 @@ static int RunModeSetLiveCaptureWorkersForDevice(ConfigIfaceThreadsCountFunc Mod
 
     /* create the threads */
     for (int thread = 0; thread < threads_count; thread++) {
+#ifdef HAVE_DPDK
+        // run only in DPDK runmode
+        if (recv_module_id == TMM_RECEIVEDPDK) {
```

Comment:
In v3 and v4 I have discouraged this usage as DPDK YAML has provision to tell which are worker thread, management thread etc. Hence please use the right formats. rework

Reply:
Code removed due to the switch from lcores to pthreads